### ISSUE TITLE
python-debian 0.1.39 does not work on Windows

### ISSUE THREAD

**Issue by carmenbianca** [fsfe/reuse-tool#311]
Example CI run: https://github.com/fsfe/reuse-tool/runs/1778624975

```
py.test
ImportError while loading conftest 'D:\a\reuse-tool\reuse-tool\tests\conftest.py'.
tests\conftest.py:23: in <module>
    from debian.copyright import Copyright
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\copyright.py:59: in <module>
    from debian import deb822
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\deb822.py:321: in <module>
    import debian.changelog
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\changelog.py:93: in <module>
    import pwd
E   ModuleNotFoundError: No module named 'pwd'
mingw32-make: *** [Makefile:69: test] Error 4
Error: Process completed with exit code 1.
```

python-debian added a dependency on `pwd`, which only exists on Unix-y systems.

**Comment by carmenbianca** [comment]
Upstream MR was already created by @rpavlik at https://salsa.debian.org/python-debian-team/python-debian/-/merge_requests/38

Thanks!

**Comment by rpavlik** [comment]
Yes, and you suggested a better patch 😄 I'll let you submit your changes on your own if you prefer, otherwise I'll revise mine to match yours.

**Comment by rpavlik** [comment]
OK, merged upstream, so version 0.1.40 of python-debian should include it
